PLAT-126566: Update kitten browser documentation (#160)
enactjs/docs: src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

{
	"name": "enact-tutorial-kitten-browser",
	"version": "1.0.0",
	"description": "A general template for an Enact Sandstone application.",
	"author": "",
	"main": "src/index.js",
	"scripts": { [omitted] },
	"license": "UNLICENSED",
	"private": true,	
	"repository": "",
	"enact": {

>         "isomorphic": true,
> 		"ri": {
> 			"baseSize": 48
> 		}
>     },

Reviewer: ```suggestion
    	"theme": "sandstone"
    },
```

Reviewer: Please check the indentation.
This should be fixed in your source code later too.

Author: OK. fixed in latest commit

`import`. We've also replaced the content with the basic markup for a single photo.
```js
import kind from '@enact/core/kind';
import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
import React from 'react';

> 	

Reviewer: Please remove a tab.

Author: OK. It's removed

			<div className={props.className}>
				<div>
					{/* For the feline-declined, you can replace the keyword below */}
					<img src="//loremflickr.com/300/300/kitten" alt="Kitten" />
					<div>Kitten</div>
				</div>
			</div>
		);
	}
});

const App = ThemeDecorator(AppBase);

export default App;
export {
	App, 
	AppBase
};
```

### JSX Hints

You might be wondering why the `<img />` tag uses the self-closing syntax (`/>`). This is a
[requirement imposed by JSX](https://facebook.github.io/react/tips/self-closing-tag.html), so
whenever you include an element that doesn't have any children, you must either use the self-closing

> syntax or explicitly close it (`<img src=""></img>`). Also when you use the `<img />` tag, it is recommended to define an alt property for accessibility and failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img).

Reviewer: ```suggestion
syntax or explicitly close it (`<img src=""></img>`). Also, when you use the `<img />` tag, it is recommended to define an alt property for accessibility failure to load the image. Detailed information is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img).
```

Reviewer: I think this one should move to somewhere else.

Reviewer: @YunhoLee0 Any ideas?

Author: I'm going to create "ESLint Hints" part including this one. Newly created part will be after "JSX Hints" part.

Author: Fixed in latest commit.

![Kitten Browser Step 1](KittenBrowser-Step1.png)
> Tell the kids to avert their eyes!

### ESLint Hints

> When you enter the above command, you'll see the compilation warning message below. Let's take a look at what this message means and try to resolve it briefly.

Reviewer: ```suggestion
When you run the above command, you'll see the compilation warning message below. Let's take a look at what this message means and try to resolve it briefly.
```

Author: OK

```console
Compiled with warnings.

./src/App/App.js
  Line 13:6:  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.
```

> The `npm run serve` compiles the source code and creates a web server. In the process of compiling the code, it uses ESLint to perform a static analysis of the code written in JavaScript, JSX. ESLint helps you to find syntax errors, anti-patterns and to write the source code in a consistent code style. For more detail is [about ESLint](https://eslint.org/docs/about/). If you only want a static analysis of the code you wrote, use `npm run lint`.

Reviewer: ```suggestion
The `npm run serve` command (aliased as `enact serve`) will build and host your project on a web server. While building the code, it allows ESLint to perform a static analysis of the source code. ESLint helps you to find syntax errors, anti-patterns, and to write the source code in a consistent code style. See [about ESLint](https://eslint.org/docs/about/) for more information. If you only want to run a static analysis of the code you wrote, use `npm run lint` command (aliased `enact lint`).
```

Author: OK

> Back in the meaning of the message, ESLint says img elements should have an `alt` property. That's right. When you use the `<img />` tag, it is recommended to define an `alt` property for accessibility failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img). We'll add the `alt` property to the tag as follows. See if it is compiled successfully.

Reviewer: ```suggestion
Returning to the warning message, ESLint says img elements should have an `alt` property. That's right. When you use the `<img />` tag, it is recommended to define an `alt` property for accessibility failure to load the image. More info is available on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img). We'll add the `alt` property to the tag as follows. See if it is built successfully.
```

Author: "Returning to the warning message" looks better than previous one.
Thank you.
It will be fixed with your suggestion.

```js
<img src="//loremflickr.com/300/300/kitten" alt="Kitten" />
```

> 

Reviewer 2: Let's move the title Conclusion here.
Change the line 135 "Conclusion" to "Serve the App" 
CC: @MikyungKim

Author: I'll check all links. That should be fixed..
Thank you.

Author: OK. "Conclusion" will be changed.
